src/views/HomeView.js

```diff
@@ -188,8 +188,11 @@ class HomeView extends AbstractView {
 
       for (const area of Helper.areas) {
         if (!Helper.strategyOptions.areas[area.area_id]?.hidden) {
+          let options = Helper.strategyOptions.areas[area.area_id ?? "undisclosed"];
+          const optionsForAllAreas = Helper.strategyOptions.areas["_"];
+          options = {...optionsForAllAreas,...options}
```

Comment:
Merge the options in a single statement

```js
options = {
  ...Helper.strategyOptions.areas["*"];
  ...Helper.strategyOptions.areas[area.area_id ?? "undisclosed"],
}

Reply:
I agree that * would be even nicer. The problem is that it is not supported by YAML, in the simple form. So I thought underscore would be the second best option. It is not very uncommon either to use underscore as wildcard in code. Examples:
* https://stackoverflow.com/questions/18300654/underscore-arrow-what-is-this
* https://stackoverflow.com/questions/41085189/using-underscore-variable-with-arrow-functions-in-es6-typescript

If you still don't like the underscore I'll adapt of course. Some other options I have been thinking about is:
```yaml
strategy:
  type: custom:mushroom-strategy
  options:
    areas:
      "*":
        type: area
        area: "{{area_id}}"
        navigation_path: "{{area_id}}"
```
```yaml
strategy:
  type: custom:mushroom-strategy
  options:
    all_areas:
      type: area
      area: "{{area_id}}"
      navigation_path: "{{area_id}}"
    areas:
      ...
```
```yaml
strategy:
  type: custom:mushroom-strategy
  options:
    areas:
      common_area_card_options:
        type: area
        area: "{{area_id}}"
        navigation_path: "{{area_id}}"
```